### PR TITLE
Docs/issue 127

### LINKED ISSUE #127: Update docs about CLI options that can be defined using files or environment variables
In the description for `--beta-build-localizations=BETA_BUILD_LOCALIZATIONS` in`docs/app-store-connect/publish.md`, minor text edits are required. Here's the edited version:
>Localized beta test info for what's new in the uploaded build as a JSON encoded list. For example, `[{"locale": "en-US", "whats_new": "What's new in English"}]`. See `--locale` for possible locale options. If not given, the value will be checked from the environment variable `APP_STORE_CONNECT_BETA_BUILD_LOCALIZATIONS`. Alternatively to entering ` BETA_BUILD_LOCALIZATIONS` in plaintext, it may also be specified using the `@env:` prefix followed by an environment variable name, or the `@file:` prefix followed by a path to the file containing the value. Example: `@env:<variable>` uses the value in the environment variable named `<variable>`, and `@file:<file_path>` uses the value from file at `<file_path>`.

Link to diff: https://github.com/codemagic-ci-cd/cli-tools/pull/124/files#r660574082

### PR DESCRIPTION
**Pull request by priitlatt**
Fixes #127.

### CODE REVIEW

File: docs/android-app-bundle/build-apks.md [hunk 1]
```diff
@@ -27,15 +27,15 @@ Path to the keystore to sign the apk files with
 ##### `--ks-pass=KEYSTORE_PASSWORD`
 
 
-Keystore password. If not given, the value will be checked from environment variable `KEYSTORE_PASSWORD`. Alternatively to entering` KEYSTORE_PASSWORD `in plaintext, it may also be specified using a `@env:` prefix followed by a environment variable name, or `@file:` prefix followed by a path to the file containing the value. Example: `@env:<variable>` uses the value in the environment variable named `<variable>`, and `@file:<file_path>` uses the value from file at `<file_path>`. [Default: None]
+Keystore password. If not given, the value will be checked from the environment variable `KEYSTORE_PASSWORD`. Alternatively to entering `KEYSTORE_PASSWORD` in plaintext, it may also be specified using the `@env:` prefix followed by an environment variable name, or the `@file:` prefix followed by a path to the file containing the value. Example: `@env:<variable>` uses the value in the environment variable named `<variable>`, and `@file:<file_path>` uses the value from file at `<file_path>`. [Default: None]
```
Comment (helinanever) [inline docs/android-app-bundle/build-apks.md:30]: missing "the":
from **the** file at `<file_path>`

Comment (helinanever) [reply docs/android-app-bundle/build-apks.md:30]: Commited the missing "the" to the respective file

Comment (priitlatt) [reply docs/android-app-bundle/build-apks.md]: Regenerated docs in a6a823a.

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**